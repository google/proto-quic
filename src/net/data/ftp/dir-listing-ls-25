drwxr-xr-x    3 ftp      ftp          4096 15 апр 18:11 .
drwxr-xr-x    3 ftp      ftp          4096 15 июл 18:11 ..
-rw-r--r--    1 ftp      ftp           528 01 май 2007 .message
-rw-r--r--    1 ftp      ftp           528 01 ноя 2007 README
-rw-r--r--    1 ftp      ftp           560 28 сен 2007 index.html
drwxr-xr-x   33 ftp      ftp          4096 12 фев 2008 pub
