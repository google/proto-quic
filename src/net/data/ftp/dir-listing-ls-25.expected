d
.
-1
1994
4
15
18
11

d
..
-1
1994
7
15
18
11

-
.message
528
2007
5
1
0
0

-
README
528
2007
11
1
0
0

-
index.html
560
2007
9
28
0
0

d
pub
-1
2008
2
12
0
0
